website/client-old/js/services/questServices.js

```diff
@@ -104,6 +104,9 @@ angular.module('habitrpg')
             Groups.data.party = party;
             $state.go('options.social.party');
             resolve();
+            if ($state.current.name === "options.social.party") {
+              $state.reload();
```

Comment:
It's been a while since I looked at the code, but is there a reason to have this logic here instead of in the controller that uses this service? Wouldn't need special logic then.